src/constants/index.ts

```diff
@@ -2,8 +2,11 @@
  * TFS centralised constants, referred as `@ukef/constants`.
  * Following constants are served:
  *
- * 1. Application
+ * 1. acbs
+ * 2. Application
+ * 3. Auth
```

Comment:
Is there a capitalisation convention we should follow here? Should it be `ACBS` or `Acbs` maybe?

Reply:
updated now